src/assessor_tools.yml

```diff
@@ -283,9 +377,22 @@
         archive_src: "https://github.com/SimplySecurity/SimplyTemplate/\
         tarball/master"
         install_dir: /tools/SimplyTemplate
+        pip_packages:
```

Comment:
Do we care about the following system Python packages their `setup.sh` script installs?

* python-tk
* python-pmw
* python-imaging

Reply:
Please see commit fdcb830.